pkg/infoschema/infoschema_v2.go

```diff
@@ -609,16 +609,10 @@ func applyAlterPolicy(b *Builder, m *meta.Meta, diff *model.SchemaDiff) ([]int64
 }
 
 func applyCreateOrAlterResourceGroup(b *Builder, m *meta.Meta, diff *model.SchemaDiff) error {
-	if b.enableV2 {
-		return b.applyCreateOrAlterResourceGroupV2(m, diff)
-	}
 	return b.applyCreateOrAlterResourceGroup(m, diff)
 }
 
 func applyDropResourceGroup(b *Builder, m *meta.Meta, diff *model.SchemaDiff) []int64 {
-	if b.enableV2 {
-		return b.applyDropResourceGroupV2(m, diff)
-	}
 	return b.applyDropResourceGroup(m, diff)
 }
```

Comment:
maybe remove the wrap function

Reply:
According to the name convertion, 

`applyDropResourceGroup(b *Builder ...)` function means it support both v1 and v2

`func (b *Builder) applyDropResourceGroup` method means it's for v1

`func (b *Builder) applyDropResourceGroupV2()` method means it's for v2